Makefile

```diff
@@ -18,29 +18,26 @@ run: generate fmt vet
 
 # Install CRDs into a cluster
 install: manifests
-	kubectl apply -f config/crds
+	kubectl apply -f config/default/crds
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy: manifests
-	kubectl apply -f config/crds
 	kustomize build config/default | kubectl apply -f -
 
 deploy-dev: manifests
-	kubectl apply -f config/crds
 	./image_patch_dev.sh
 	kustomize build config/overlays/development | kubectl apply -f -
 
-undeploy: manifests
+undeploy:
 	kustomize build config/default | kubectl delete -f -
-	kubectl delete -f config/crds
 
-undeploy-dev: manifests
+undeploy-dev:
 	kustomize build config/overlays/development | kubectl delete -f -
-	kubectl delete -f config/crds
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests:
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
+	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd --output-dir=config/default/crds
+	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go rbac --output-dir=config/default/rbac
```

Comment:
Do we also need to generate manager manifests ? Any reason why you do not want to do all ?

Reply:
`go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all` only generate `crds` and `brace`